### PR TITLE
Ticket 17 revisited: More control over CN-ID redaction.

### PR DESCRIPTION
**Pull request by robstradling**

### CODE REVIEW

File: rfc6962-bis.xml [hunk 1]
```diff
@@ -348,19 +348,23 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
...
+          </t>
+        </section>
+        <section title="Redacted Labels Certificate Extension" anchor="redaction_extension">
+          <t>
+            When a precertificate contains one or more <spanx style="verb">?</spanx> labels, a non-critical extension (OID 1.3.101.77, whose extnValue OCTET STRING contains an ASN.1 SEQUENCE OF INTEGERs) MUST be added to the corresponding certificate. The purpose of this extension is to enable TLS clients to reconstruct the TBSCertificate component of the precertificate from the certificate, as described in <xref target="reconstructing_tbscertificate"/>.
+          </t>
+          <t>
+            Each INTEGER MUST have a value of zero or more. The first INTEGER indicates the total number of <spanx style="verb">?</spanx> labels in the precertificate's first DNS-ID; the second INTEGER does the same for the precertificate's second DNS-ID; etc. The last INTEGER does the same for the precertificate's zero or more CN-IDs. There MUST NOT be more INTEGERs than there are DNS-IDs plus one; if there are fewer INTEGERs than this, the shortfall is made up by implicitly repeating the last INTEGER.
```
Comment (eranmes) [inline rfc6962-bis.xml]: Can you be explicit about the number of expected labels in the case of missing CN-ID?

Comment (eranmes) [inline rfc6962-bis.xml]: I suggest:
"than there are DNS-IDs plus one" -> "than there are DNS-IDs plus one for all CN-IDs"

Comment (robstradling) [inline rfc6962-bis.xml]: How about:
"There MUST NOT be more INTEGERs than there are DNS-IDs (plus one, if any CN-IDs are present)"
?

Comment (eranmes) [inline rfc6962-bis.xml]: Much better than my suggestion :)

**Comment by robstradling** [comment]
Merged at https://github.com/google/certificate-transparency-rfcs/commit/11b145a7c9865dd8e6f04c6edb19e6fdc4b1a174

**Pull request closed without merging**